packages/assistant-stream/src/core/utils/stream/AssistantMetaTransformStream.ts

```
@@ -0,0 +1,72 @@
+          chunk.type === "text-delta" ||
+          chunk.type === "result" ||
+          chunk.type === "part-finish" ||
+          chunk.type === "tool-call-args-text-finish"
+        ) {
+          if (chunk.path.length !== 1) {
+            controller.error(
+              new Error(`${chunk.type} chunks must have a path of length 1`),
+            );
+            return;
+          }
+          const idx = chunk.path[0]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)